services/Service.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const apiClient = axios.create({
+  baseURL: `http://localhost:3000`,
```

Comment:
Tutaj jest możliwość wykorzystania w konfiguracji axiosa zmienne środowiskowe i tym samym utworzyć środowiska które np. uderzają do innego hosta. Jest to użyteczne w przypadku kilku środowisk np. lokalne, developerskie, testowe, produkcyjne. W formie zajawki zostawiam pattern do zapoznania - git flow :).

Niestety nie wiem jak w środowisku vue można takie zmienne zaimplementować. W webpacku jest define plugin, ale musiałbyś ejectować projekt i wymaga wiedzy z webpacka. Pewnie da się skonfigurować vue cli, aby to osiągnąc. Do dooguglania :).